src/pybind/mgr/dashboard/tests/test_osd.py

```diff
@@ -0,0 +1,240 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import uuid
+from contextlib import contextmanager
+
+from mock import patch
+
+from .helper import ControllerTestCase
+from ..controllers.osd import Osd
+from .. import mgr
+from ..tools import update_dict
+
+try:
+    from typing import List, Dict, Any  # pylint: disable=unused-import
+except ImportError:
+    pass  # Only requried for type hints
+
+
+class OsdHelper(object):
```

Comment:
Great to have some unit-test for this!

Reply:
:smile: I feel the same way. Unit-testing OSDs should also be easier as of now.